### ISSUE TITLE
TypeError: 'NoneType' object is not iterable for control panel

### ISSUE THREAD

**Issue by AritroSaha10** [uvjustin/alarmdotcom#108]
Here's the full error:

```
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 368, in async_add_entities
tasks = [
File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 368, in <listcomp>
tasks = [
File "/config/custom_components/alarmdotcom/alarm_control_panel.py", line 85, in <genexpr>
ADCIControlPanel(
File "/config/custom_components/alarmdotcom/alarm_control_panel.py", line 106, in __init__
self._conf_arm_home: set = set(
TypeError: 'NoneType' object is not iterable
```

This leads to the partition device only having a Malfunction entity, but no alarm control panel. Seems like this is due to a recent change as well, as it seems to be working in v0.2.1 ([reference code if wanted](https://github.com/aritrosaha10/alarmdotcom))

[LOC ref if useful](https://github.com/uvjustin/alarmdotcom/blob/master/custom_components/alarmdotcom/alarm_control_panel.py#L106)

**Comment by elahd** [comment]
Do you see a partition if you run adc with just the username and password parameters?

**Comment by AritroSaha10** [comment]
Yeah, looks to be normal:

```
*** PARTITIONS ***

--------
partition1 (102857060-127)
        State: DeviceState.DISARMED 
        Battery: Normal
--------
```

**Comment by elahd** [comment]
Should be fixed in v0.2.3-beta.3. Would you mind testing?

**Comment by AritroSaha10** [comment]
Works now!
